buildSrc/src/main/groovy/com/uber/okbuck/core/model/android/AndroidTarget.groovy

```diff
@@ -240,14 +240,16 @@ abstract class AndroidTarget extends JavaLibTarget {
     }
 
     String processManifestXml(GPathResult manifestXml) {
-        def sdkNode = {
-            'uses-sdk'('android:minSdkVersion': String.valueOf(minSdk),
-                    'android:targetSdkVersion': String.valueOf(targetSdk)) {}
-        }
         if (manifestXml.'uses-sdk'.size() == 0) {
-            manifestXml.appendNode(sdkNode)
+            manifestXml.appendNode {
+                'uses-sdk'('android:minSdkVersion': String.valueOf(minSdk), 'android:targetSdkVersion': String.valueOf(targetSdk)) {}
+            }
         } else {
-            manifestXml.'uses-sdk'.replaceNode(sdkNode)
+            def overrideLibrary = manifestXml.'uses-sdk'.@'tools:overrideLibrary'
```

Comment:
Can you add this only if it is declared/exists?

This should be explicitly declared in the base manifest before we can add it